### PR TITLE
Use pattern matching, proper casing, whitespace cleanup, etc.

### PR DESCRIPTION
**Pull request by codemonkey85**
Nothing too wild here, but I did think pattern matching was more readable in places.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-$on(window, 'hashchange', onHashchange);
+import { dotnet } from "./dotnet.js"
+import { $on } from "./helpers.js";
 
 const { getAssemblyExports, getConfig } = await dotnet.create();
 
-exports = await getAssemblyExports(getConfig().mainAssemblyName);
+let exports = await getAssemblyExports(getConfig().mainAssemblyName);
+
+debugger;
```
Comment (pavelsavara) [inline main.js]: please remove `debugger`

File: view.js [hunk 2]
```diff
@@ -12,24 +12,24 @@ let $toggleAll;
...
 export function editItem(target) {
 	const listItem = target.parentElement.parentElement;
 
-	listItem.classList.add('editing');
+	listItem.classList.add("editing");
 
-	const input = document.createElement('input');
-	input.className = 'edit';
+	const input = window.document.createElement("input");
+	input.className = "edit";
```
Comment (pavelsavara) [inline view.js]: let's revert all quote changes in the JS files, so that it matches the original as much as possible.

File: main.js (hunk 1 above)
Comment (codemonkey85) [reply main.js]: Ahhh, whoops! 🤦🏻‍♂️

File: view.js (hunk 2 above)
Comment (codemonkey85) [reply view.js]: Gotcha. That was a ReSharper suggestion.

**Comment by codemonkey85** [comment]
Fixes applied.

File: view.js [hunk 3]
```diff
@@ -66,15 +66,15 @@ export function setCompleteAllCheckbox(checked) {
 
 export function updateFilterButtons(route) {
 	qs('.filters .selected').className = '';
-	qs(`.filters [href="#/${route}"]`).className = 'selected';
+	qs(`.filters [href='#/${route}']`).className = 'selected';
```
Comment (pavelsavara) [inline view.js:69]: quotes in the xml are ok. again, I prefer minimal change

File: Program.cs [hunk 4]
```diff
@@ -1,33 +1,25 @@
...
+        var store = new Store();
+        var view = new View(new Template());
+        _controller = new Controller(store, view);
+        Console.WriteLine("Ready!");
     }
-}
\ No newline at end of file
+
+    [JSExport]
+    public static void OnHashchange(string url) => _controller?.SetView(url);
```
Comment (pavelsavara) [inline Program.cs:24]: I think there was a problem with arrow function JSExport in RC1, but I'm not sure. Does it work with RC1 ?

Comment (codemonkey85) [reply Program.cs:24]: Not sure, I tested this on RC2.

Comment (pavelsavara) [reply Program.cs:24]: Let's not make that change please

File: view.js (hunk 3 above)
Comment (codemonkey85) [reply view.js:69]: Just wondering, why have inconsistent quotes? Why not all single or all double?

**Pull request closed without merging**